src/comm_manager.cpp

```diff
@@ -526,6 +534,23 @@ void CommManager::send_named_value_float(const char * const name, float value)
 
 void CommManager::send_next_param(void)
 {
+  //PTT Hack
```

Comment:
Should we figure out a more permanent solution for this before merging these changes into main?

Reply:
yes we should think of a more permanent solution